statev2/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "statev2"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+
+# === Storage === #
+flexbuffers = "2.0"
+libmdbx = "0.3"
```

Comment:
idk if it's relevant to us, but reth forked an earlier version of this crate b/c it switched to GPL license: https://github.com/paradigmxyz/reth/tree/main/crates/storage/libmdbx-rs

Reply:
yeah planning to look into all that later